app/src/main/java/org/httpsknighthacks/knighthacksandroid/Workshops.java

```diff
@@ -25,6 +25,8 @@
 
 public class Workshops extends AppCompatActivity {
 
+    private static final String allFILTER = "All";
```

Comment:
Let's make the names of these String variables all caps to indicate they are  static final.